_config.yml

```diff
@@ -63,21 +63,15 @@ logo:
   subtitle: ''
 
 nav_links:
-  Meteor jobs:
-    url: https://www.cultofcoders.com
-    taget: _new
-  Atmosphere:
-    url: https://www.atmospherejs.com
-    target: _new
-  NPM:
-    url: https://www.npmjs.com/
-    target: _new
   Meteor API:
     url: https://docs.meteor.com/
     target: _new
-  Meteor Guide:
-    url: https://guide.meteor.com/
+  Github repository:
```

Comment:
It doesn't make sense to put it here. This is only for Chapter 1.

Reply:
It is for chapter 1 & 2 if you take a look more careful. It also has routes / login / register + basic design structure

Reply:
It still doesn't make sense to put it here. Put it inside Chapter 1. And make it very evident. + it occupies too much space "Github repository" doesn't really say anything.